rollup.config.js

```diff
@@ -80,16 +87,35 @@ const prodUmdConfig = Object.assign({}, umdConfig, {
   ]),
```

Comment:
The UMDs above don't need the streaming support either so we can set `__SERVER__` to false there as well :) it'd be great to just have browser stuff in those

Reply:
It is: https://github.com/styled-components/styled-components/pull/1430/files#diff-ff6e5f22a9c7e66987b19c0199636480R56

Reply:
https://github.com/styled-components/styled-components/pull/1430/files#diff-ff6e5f22a9c7e66987b19c0199636480R53